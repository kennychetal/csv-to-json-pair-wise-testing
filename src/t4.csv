letter,number,symbol,
j,3,@,
m,4,$,
a,7,%,