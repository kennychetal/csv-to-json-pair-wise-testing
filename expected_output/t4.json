[
    {
        "symbol": "@", 
        "number": "3", 
        "letter": "j"
    }, 
    {
        "symbol": "$", 
        "number": "4", 
        "letter": "m"
    }, 
    {
        "symbol": "%", 
        "number": "7", 
        "letter": "a"
    }
]